frontend/src/helpers/uuid/get-uuid/get-uuid.helper.ts

```diff
@@ -0,0 +1,7 @@
+import { v4 as uuid } from 'uuid';
+
+const getUuid = (): string => {
```

Comment:
```suggestion
const getRandomId = (): string => {
```
don't forget to rename the file
and move it to the `string` helpers